talent/job-descriptions/offers/applications-pm-sf.md

```diff
@@ -0,0 +1,30 @@
+Senior Product Manager in the Applications team
+------------------------------------------
+
+### [Location & Seniority](../sf-location-seniority-section.md)
+
+**Compensation:** competitive market salary + 0.1% [stock options](https://github.com/src-d/guide/blob/master/talent/esop.md)
+
+### [Company](../company-section.md)
+
+### [Engineering at source{d}](../engineering-section.md)
+
+### Role
+
+This position is for a Senior Product Manager on the Applications team. 
```

Comment:
Let's just say Senior Product Manager at source{d} so we don't have to confuse people yet

Reply:
It is for the applications team but I rather explain that in the interview process itself, since applications could mean mobile or other things to other people.